m1.lesson1.web_calculator/calc.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<head>
+        <style>
+                .button {
+    
+                    background-color: #4CAF50;
+                    width: 200px;
+                    color: black;
+                    padding: 15px 32px;
+                    font-size: 16px;
+                    margin: 4px 2px;
+                    cursor: pointer;
+                    border-radius: 100%;
+                    
+                }
+                </style>
+<center><h1>Calculater</h1></center>

```

Comment:
do not use element "center" as it is obsolete
see
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center